cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casAccountDisabledView.jsp

```diff
@@ -28,6 +28,14 @@
     <p><spring:message code="screen.accountdisabled.message"/></p>
 </div>
 
+<spring:message code="screen.osf.login.message.error" var="errorDescription"/>
+<script>
+    description = document.getElementById("description");
```

Comment:
Unlike `bottom.js`, `<c:set var="..." value="..."/>` does not work for `top.js`. Alternatively, use a script to update the title description in error pages.